packages/react-fiche-service-public/src/components/Table.js

```diff
@@ -27,35 +27,50 @@ class Table extends React.PureComponent {
       return columns[columnIndex]._.type === ROW_HEADER;
     };
 
+    const handleSpan = el => {
+      let colspan = 1;
+      let rowspan = 1;
+      if (el._) {
+        colspan = el._.fusionHorizontale || 1;
+        rowspan = el._.fusionVerticale || 1;
+      }
+      return {
+        colspan,
+        rowspan
+      };
+    };
+
     return (
       <table>
         {title && <caption>{getText(title)}</caption>}
-        {headingRow && (
+        {headingRows.length && (
```

Comment:
`> 0`

Reply:
`0` will evaluate to false so it's ok I guess

Reply:
thanks :) I know but we try to avoid type coercion and prefer being explicit

Reply:
We shall be more consistent then because you used the .length without `> 0` on the theme page